docs/even-odd.md

```diff
@@ -0,0 +1,22 @@
+# Even Odd
+
+An even number is an integer which is "evenly divisible" by two. This
+means that if the integer is divided by 2, it yields no remainder.
+
+An odd number is an integer which is not evenly divisible by two. This
+means that if the integer is divided by 2, it yields a remainder of 1.
+
+## Requirements
+
+Create a file called Even Odd using the naming
+convention appropriate for your language of choice.
+
+Write a sample program which accepts an integer on the command line and
+outputs if the integer is Even or Odd.
+
+## Testing
+
+Verify that the actual output matches the expected output. See the
```

Comment:
I think for both this and prime you could include a table of example inputs and outputs like reverse a string or roman numeral. That way we can explore all the possible test cases. This one should include one odd number, one even number, zero, and a negative number.